ext/phar/tests/bug45218_SLOWTEST.phpt

```diff
@@ -19,21 +19,23 @@ class myIterator implements Iterator
     var $a;
     var $count = 1;
 
+    #[ReturnTypeWillChange]
```

Comment:
Something weird is going on here, the test timed out if I made this method and `rewind` void

Reply:
Works fine for me.